src/Lykke.Service.PayInternal/Modules/ServiceModule.cs

```diff
@@ -79,6 +82,10 @@ private void RegisterAzureRepositories(ContainerBuilder builder)
                 AzureTableStorage<BlockchainTransactionEntity>.Create(
                     _dbSettings.ConnectionString(x => x.MerchantConnString),
                     "MerchantWalletTransactions", _log)));
+
+            builder.RegisterInstance<ITransferRepository>(new TransferRepository(
```

Comment:
For repository you can register type, but not instance. Better register instance of INoSQLTableStorage